test/validators.js

```diff
@@ -4276,6 +4276,8 @@ describe('Validators', () => {
         'rgba(255,255,255,0.1)',
         'rgb(5%,5%,5%)',
         'rgba(5%,5%,5%,.3)',
+        'rgb( 255,255 ,255)',
+        'rgba(255, 255, 255, 0.5)',
       ],
```

Comment:
I Think it's worthwhile to add a test case like this for clarity's sake:
 
```suggestion
        'rgba(255, 255, 255, 0.5)',
        'r         g    b(   0,         251,       222     )',
      ],
```

Reply:
changed so that it does not ignore spaces between rgb at start, and added the test cases